tests/k8s-azure/manifest/linux-kcm.json

```diff
@@ -7,6 +7,7 @@
             "orchestratorRelease": "1.14",
             "kubernetesConfig": {
                 "customHyperkubeImage": "gcrio.azureedge.net/google_containers/hyperkube-amd64:v1.13.0-alpha.3",
+                "privateAzureRegistryServer": "gcrio.azureedge.net",
```

Comment:
actually it's gcr.io, not azure registry...

Reply:
@andyzhangx This value will be replaced by the generated image that is being pushed to ACR.